packages/styles/scss/internal/scroll-into-view/_scroll-into-view.scss

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Not sure if this file technically belongs in `internal`, maybe move to `components` since it is part of a component now. Be sure to update corresponding documentation too.